<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>107</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>RSA file encryption</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>494</width>
     <height>83</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="inputLabel">
      <property name="text">
       <string>Input file:</string>
      </property>
      <property name="buddy">
       <cstring>inputLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="keyLabel">
        <property name="text">
         <string>Key:</string>
        </property>
        <property name="buddy">
         <cstring>keyEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="keyEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="3" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout2">
      <item>
       <widget class="QRadioButton" name="encryptRadioButton">
        <property name="text">
         <string>Encrypt</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="decryptRadioButton">
        <property name="text">
         <string>Decrypt</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="6">
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QPushButton" name="inputBrowse">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QPushButton" name="outputBrowse">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="5">
     <widget class="QLineEdit" name="inputLineEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="outputLabel">
      <property name="text">
       <string>Output file:</string>
      </property>
      <property name="buddy">
       <cstring>outputLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="5">
     <widget class="QLineEdit" name="outputLineEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>inputLineEdit</tabstop>
  <tabstop>inputBrowse</tabstop>
  <tabstop>outputLineEdit</tabstop>
  <tabstop>outputBrowse</tabstop>
  <tabstop>keyEdit</tabstop>
  <tabstop>encryptRadioButton</tabstop>
  <tabstop>decryptRadioButton</tabstop>
  <tabstop>okButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>encryptRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainDialog</receiver>
   <slot>encryptRadioButtonToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inputBrowse</sender>
   <signal>clicked()</signal>
   <receiver>MainDialog</receiver>
   <slot>inputBrowseButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>outputBrowse</sender>
   <signal>clicked()</signal>
   <receiver>MainDialog</receiver>
   <slot>outputBrowseButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>505</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>MainDialog</receiver>
   <slot>okButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>encryptRadioButtonToggled(bool)</slot>
  <slot>inputBrowseButtonClicked()</slot>
  <slot>outputBrowseButtonClicked()</slot>
  <slot>okButtonClicked()</slot>
 </slots>
</ui>
